requirements-dev.txt

```diff
@@ -6,4 +6,5 @@ responses
 backports.tempfile
 mock
 unittest2
+sphinx-argparse
```

Comment:
`requirements-dev` is for test/development-only requirements, but you are importing these packages in the main codebase. So you have to put this in `requirements.txt`, not `requirements-dev`.

Also, when I installed this, it didn't seem to automatically pull in sphinx, and I got an error:
```
>make docs
/Library/Developer/CommandLineTools/usr/bin/make -C docs html
Running Sphinx v1.5.5

Exception occurred:
  File "/Users/akislyuk/projects/dcp-cli/hca/util/__init__.py", line 100, in <module>
    from sphinx.parsers import RSTParser
ImportError: cannot import name RSTParser
```
I ran `pip install sphinx` and the error went away. Can you check if `sphinx-argparse` pulls in `sphinx`? If not, you have to list it here.

Reply:
added docutils to requirements.txt, added sphinx and sphinx-argparser to requirement-dev.txt